woocommerce/woocommerce
# Revert Product Editor UI Polish

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR reverts the changes from the following PRs:
 - https://github.com/woocommerce/woocommerce/pull/53121
 - https://github.com/woocommerce/woocommerce/pull/52632

As we're reluctant to release such changes at this time of year and would like to assert better extension compatibility (see the internal discussion pf59Ax-2vk-p2#comment-1994 )



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout and build this branch
2. Preferably open another test site side by side with WC 9.5, or WC at 480c111c11de671dc1ed607b1d0bafc685763bca
5. Install a few canonical extensions (p6q6po-e59-p2), like AutomateWoo, Google for WooCommerce, WooCommerce Subscriptions, Product Bundles, etc.
6. Open the updated pages:
   - "Add product": Products > Add new product
   - "Edit product": Products > All products > (pick one) > edit
   - "Add coupon": Marketing > Coupons >Add new Coupon
   - "Edit coupon": Marketing > Coupons >(pick one) > edit
 7. Check that it looks and behaves as before (480c111c11de671dc1ed607b1d0bafc685763bca)
 8. Browse through the view switch tabs in editor view, product/coupon-data box, collapse meta boxes, add/remove images, etc.
 9. Resize the window to smaller one, to check the responsiveness
 10. Disable JavaScript (open browser dev tools (F12) > ctrl+shift+p > type "javascript"), reload the page, test the page in no-js mode.
 
![image](https://github.com/user-attachments/assets/c5aa6547-a37f-4888-8ba9-934e675cc0c7)




### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
…upon editor (#53121)"

This reverts commit bca2df3.
